tools/flang2/flang2exe/aarch64-Linux/machreg.h

```diff
@@ -353,12 +335,47 @@ typedef struct {
 extern REG reg[];
 extern RGSETB rgsetb;
 
+#define SCRATCH_REGS {IR_RAX, IR_RCX, IR_RDX}
+
+#define MACH_REGS \
+  { \
+    {1, 8, 8 /*TBD*/, MR_L1, MR_U1, MR_U1, MR_U1, 0, 0, 'i'},       /*  %r's  */ \
+    {1, 8, 8 /*TBD*/, MR_L2, MR_U2, MR_U2, MR_U2, 0, MR_MAX1, 'f'}, /*  %f's  */ \
+    {1, 8, 8 /*TBD*/, MR_L3, MR_U3, MR_U3, MR_U3, 0, (MR_MAX1 + MR_MAX2), \
+     'x'} /*  %f's  xmm */ \
+  }
```

Comment:
@pawosm-arm Are you planning more work in this file? These structure definitions aren't updated for AArch64 yet.

Reply:
Sadly, as we didn't observe performance impact of this file, further work on this was halted long time ago which remained this change purely a conceptual work.